tools/ci/install-pkgdown-dependencies.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+# Context: .github/workflows/*.yaml
+export R_REMOTES_NO_ERRORS_FROM_WARNINGS='true'
+Rscript -e 'install.packages("remotes")'
+Rscript -e 'install.packages("pkgdown")'
```

Comment:
That is style wart:   `Rscript -e 'install.packages(c("remotes", "pkgdown"))'`  is more compact.